pyproject.toml

```diff
@@ -85,8 +85,4 @@ enable_error_code = [
     "redundant-expr",
     "truthy-bool",
 ]
-show_error_codes = true
 warn_unreachable = true
-
-[project.entry-points."beamlime.stateless"]
-ess-loki = "ess.loki.workflow:live_workflow"
```

Comment:
Hmm, this is interesting behaviour of copier update. Need to look into this later.

Reply:
Yeah this shouldn't be deleted unless it was deleted in the template as well.
Was this accidentally added and then removed...?